package.json

```diff
@@ -59,6 +59,7 @@
     }
   },
   "dependencies": {
+    "@babel/core": "7.12.3",
```

Comment:
This jsx plugin should use `@babel/core` on user side so we don't want it to become dependency.

We can add the meta field to package.json to declare it as optional

```json
"peerDependenciesMeta": {
  "@babel/core": {
    "optional": true
  }
}
```

Reply:
Thanks for the, did not knew this option existed.